tests/end_to_end_test/end_to_end_test.go

```diff
@@ -369,27 +370,43 @@ func TestSnapshotRestore(t *testing.T) {
 	// Attempt to restore a snapshot from an empty repo.
 	e.runAndExpectFailure(t, "restore", "kffbb7c28ea6c34d6cbe555d1cf80faa9", "r1")
 	e.runAndExpectSuccess(t, "snapshot", "create", source)
-	o := e.runAndExpectSuccess(t, "snapshot", "list")
 
 	// obtain snapshot root id and use it for restore
-	root := getLastSnapshotRootID(t, o)
```

Comment:
is `getLastSnapshotRootID` still used elsewhere?
If not, nuke it.

Reply:
nuked